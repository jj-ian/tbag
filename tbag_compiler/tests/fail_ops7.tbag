/* Authors: Maria */
true {
  printLine("hello" ~~ 1);
  endgame;
}

func int printLine(boolean b) {
  print(b);
  print("\n");
  return 0;
}
