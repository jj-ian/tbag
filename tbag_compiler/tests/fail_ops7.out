Fatal error: exception Failure("Types to ~~ must both be String")
